test/parallel/test-process-raw-debug.js

```diff
@@ -63,10 +63,7 @@ function child() {
     throw new Error('No ticking!');
   };
 
-  const stderr = process.stderr;
-  stderr.write = function() {
-    throw new Error('No writing to stderr!');
-  };
+  common.hijackStderr(common.mustNotCall('stderr.write mustn\'t be called.'));
```

Comment:
Replace `'stderr.write mustn\'t be called.'` with either:

`'stderr.write must not be called.'`
or
<pre>`stderr.write mustn't be called.`</pre>